benchmarks/htap/lib/controller.py

```diff
@@ -10,9 +10,10 @@
 from psycopg2.errors import DuplicateDatabase, DuplicateTable, ProgrammingError
 
 from benchmarks.htap.lib.helpers import nullcontext
-from benchmarks.htap.lib.monitoring import Stats, Monitor
-from benchmarks.htap.lib.queries import Queries
-from benchmarks.htap.lib.transactions import Transactions
+from benchmarks.htap.lib.monitoring import Monitor
+from benchmarks.htap.lib.stats import Stats
+from benchmarks.htap.lib.analytical import AnalyticalStream
```

Comment:
why is this a stream and the other thing a worker?

Reply:
One class seems to represent a specific OLAP stream with a specific query order (gets a stream_id during creation), the transaction worker seems that all to the same workload (but random, and gets a worker_id) 
We could call both workers, I do not mind